robusta_krr/core/models/config.py

```diff
@@ -16,6 +16,7 @@ class Config(pd.BaseSettings):
     clusters: Union[list[str], Literal["*"], None] = None
     kubeconfig: Optional[str] = None
     namespaces: Union[list[str], Literal["*"]] = pd.Field("*")
+    discovery_method: Union[Literal["api-server"], Literal["prometheus"]] = pd.Field("api-server")
```

Comment:
You should be able to do like that:

`Literal["api-server", "prometheus"]`

Reply:
Fixed